### ISSUE TITLE
Image width not getting stretched

### ISSUE THREAD

**Issue by sarathvad1811** [ZXVentures/react-native-async-image-animated#5]
I absolutely love your work... but one problem I want the image to be stretched to full screen...

The placeholder fits well but the image after loading even though it's width is greater than the window's width... I saw the source code where the resizeMode is "contain" in all cases... in some cases you would want the "cover" value.... 

 Please help!!!! Thanks in advance

**Comment by peppermints** [comment]
Modify the AsyncImageAnimated.js  source code, like this：
`

        {!failed &&
            <react_native_1.Animated.Image key={imageKey} source={source} resizeMode={'contain'} style={[
                style,
                {
                    opacity: imageOpacity,
                    position: 'absolute',
                    resizeMode: 'cover',
                },
            ]} onLoad={this.onLoad} onError={this.onError}/>}

        {(placeholderSource && !loaded) &&
            <react_native_1.Animated.Image source={placeholderSource} style={[
                style,
                {
                    opacity: placeholderOpacity,
                    position: 'absolute',
                    resizeMode: 'cover',
                },
            ]}/>}}/>}
`

**Comment by alexborton** [comment]
Can this be made a prop? `resizeMode`

**Comment by sarathvad1811** [comment]
Yes please ... it can be made a prop... that would perfectly solve my problem @alexborton

**Comment by geminiyellow** [comment]
hey yo, @wmcbain , please add `resizeMode` to prop.
and, could you set the height/width to option?
you can give it '100%' default value.

**Comment by wmcbain** [comment]
@geminiyellow Will go through this weekend and make requested changes from everyone. Or feel free to submit a PR if you need it earlier.

**Comment by geminiyellow** [comment]
hmm... ok, 1 month later ...

**Comment by wmcbain** [comment]
@geminiyellow Feel free to open a PR thanks. :)

**Comment by drewdulz** [comment]
I made a PR. https://github.com/ZXVentures/react-native-async-image-animated/pull/6
